frontend/packages/design-system/src/themes/Link.ts

```diff
@@ -0,0 +1,17 @@
+export default {
+  defaultProps: {
+    isUnderlined: false,
+  },
+  baseStyle: {
+    _text: {
+      fontSize: 'xs',
+      fontWeight: 'medium',
+      color: 'secondary.500',
+    },
+    _hover: {
+      _text: {
+        color: 'primary.500',
```

Comment:
I assumed the active link would also be orange

Reply:
`_active` and `_focus` don't seem to do anything